File: kyonggiyo-core/src/main/java/kyonggiyo/domain/auth/util/PlatformConverter.java
package kyonggiyo.domain.auth.util;

import kyonggiyo.domain.auth.Platform;
import org.springframework.core.convert.converter.Converter;

public class PlatformConverter implements Converter<String, Platform> {

    @Override
    public Platform convert(String platform) {
        return Platform.from(platform);
    }

}

File: kyonggiyo-core/src/main/java/kyonggiyo/application/auth/domain/util/PlatformConverter.kt
package kyonggiyo.application.auth.domain.util;

import kyonggiyo.application.auth.domain.vo.Platform;
import org.springframework.core.convert.converter.Converter;

public class PlatformConverter implements Converter<String, Platform> {

    @Override
    public Platform convert(String platform) {
        return Platform.from(platform);
    }

}
